### PR TITLE
Update Teclis

### PR DESCRIPTION
**Pull request by Spandamn**

### CODE REVIEW

File: data/mods/ssb/moves.ts [hunk 1]
```diff
@@ -4744,32 +4744,66 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+				if (pokemon.isGrounded() && !pokemon.isSemiInvulnerable()) {
+					if (pokemon && !pokemon.hasType('Fire')) {
+						this.damage(pokemon.baseMaxhp / 8, pokemon);
+					} else if (pokemon) {
+						this.heal(pokemon.baseMaxhp / 8, pokemon, pokemon);
+					}
+				}
+			},
+			onEnd() {
+				this.add('-fieldend', 'move: Fire Pledge');
```
Comment (ghost) [inline data/mods/ssb/moves.ts:4814]: :I

Comment (Spandamn) [reply data/mods/ssb/moves.ts:4814]: Thoight it would look cool

Comment (ghost) [reply data/mods/ssb/moves.ts:4814]: Fair point,

**Comment by ghost** [comment]
Maybe we should make fiery terrain its own move.

**Comment by ghost** [comment]
ok yeah do what i said above (terrain its own move) then ill merge

**Pull request merged**